tests/metrics_test.go

```diff
@@ -153,14 +157,99 @@ var _ = Describe("Metrics", func() {
 		)
 	})
 
+	Context("SSP metrics", func() {
+		var originalVersion string
+
+		getDeployment := func(name string, namespace string) *apps.Deployment {
+			deployment := &apps.Deployment{}
+			err := apiClient.Get(ctx, client.ObjectKey{Name: name, Namespace: namespace}, deployment)
+			Expect(err).ToNot(HaveOccurred())
+			return deployment
+		}
+
+		updateVersion := func(version string) {
+			Eventually(func() error {
+				deployment := getDeployment(sspDeploymentName, sspDeploymentNamespace)
+
+				envs := deployment.Spec.Template.Spec.Containers[0].Env
+
+				for i, env := range envs {
+					if env.Name == common.OperatorVersionKey {
+						envs[i].Value = version
+						break
+					}
+				}
+
+				return apiClient.Update(ctx, deployment)
+			}, 1*time.Minute, time.Second).ShouldNot(HaveOccurred())
+		}
+
+		waitForVersionUpdate := func(version string) {
```

Comment:
This functionality can be moved into `updateVersion()` function, because it is always useful to wait until containers with new version are deployed.

Reply:
moved